src/install

```diff
@@ -25,6 +25,19 @@
         (delete-file to))
       (copy-file from to))))
 
+(define (patch-gxc-shebang)
+  (let* ((filename    (string-append default-gerbil-home "/bin/gxc"))
+         (new-content (with-input-from-file filename
+                        (lambda ()
+                          (with-output-to-string
+                            (lambda ()
+                              (read-line)
+                              (display (string-append "#!" default-gerbil-home "/bin/gxi\n"))
+                              (display (read-line (current-input-port) #f))))))))
```

Comment:
I think this may need a custom readtable to read `gxc` as it uses `[]` for list constructions.

Reply:
oh it reads a string, never mind!